src/stories/common.tsx

```diff
@@ -185,31 +185,37 @@ const SIDEBAR_WIDTH = 210;
 const SPACER = 550;
 
 const StyledForm = styled(OriginalFrontendEngine)`
-	width: calc(${V2_MediaWidths.desktopL}px - ${MINIMUM_SIDE_PADDING + SIDEBAR_WIDTH + SPACER}px);
+	width: calc(
+		${({ theme }) => Breakpoint["xl-max"]({ theme })}px - ${MINIMUM_SIDE_PADDING + SIDEBAR_WIDTH + SPACER}px
```

Comment:
can directly set the token without wrapping in a function within the top-level of styled css. update other instances on this page

```suggestion
		${Breakpoint["xl-max"]}px - ${MINIMUM_SIDE_PADDING + SIDEBAR_WIDTH + SPACER}px
```